src/ShaderManager.js

```diff
@@ -158,6 +158,11 @@ ShaderManager.DRAW_MODE = {
      */
     default: 'default',
```

Comment:
Just to clarify the situation, could you add to the comment on the default draw mode to indicate that its output uses premultiplied alpha?